### DISCUSSION TITLE
what is the relationship between Tailwind css and windi css?

### DISCUSSION THREAD

**Discussion by JenuelDev** [tailwindlabs/tailwindcss#3883] [Help]
they work the same, the new JIT was already a feature on windi css. I just want to know the relationship and what is better to use?

**Comment by ericlbarnes** [comment]
Windi is a fork of Tailwind and they seem to be going in a different direction than Tailwind. They both have a JIT compiler, and from what I've read Taildwind's is actually faster, plus it's first-party so it'll always work with the latest version of Tailwind. 

Personally, I'd recommend going with the official Tailwind version because I know it's going to work and be supported.

**Reply by JenuelDev** [reply]
ok thanks for the answer, maybe ill try them both,, thanks thanks :)

**Comment by jenstornell** [comment]
There seems to be a Tailwind Windi war going on.

In short it seems like Windi came up with the JIT idea and expected to get more credit for it from the Tailwind crew. On the other hand, Windi itself it like a rip of the whole Tailwind idea.

Feel free to correct me if I'm wrong.

https://twitter.com/antfu7/status/1371533878800748545
